src/components/Toolbar.tsx

```diff
@@ -94,12 +100,17 @@ const Toolbar: FC<ToolbarProps> = ({ customize, onSelect, selected }) => {
     }
   }, [isDraggingAny, setPressingToolbarId])
 
+  /** Handle all mouse up events. */
   useEffect(() => {
-    /**
-     * Update isPressed when the drag is released outside of the toolbar.
-     * */
+    /** Method to handle the release of the mouse. */
     const handleMouseUp = event => {
-      setPressingToolbarId(null)
+      setTimeout(
+        () => {
+          setPressingToolbarId(null)
+          onSelect?.(shortcutById(id))
```

Comment:
`onSelect` should only be called when selecting a toolbar button, not deselecting.

Reply:
I added it only because it was there before; removed